openfl/component/collaborator/collaborator.py

```diff
@@ -123,8 +126,7 @@ def __init__(
         self.delta_updates = delta_updates
 
         self.client = client
-        # Flag can be enabled to get memory usage details for ubuntu system
-        self.log_memory_usage = log_memory_usage
+
         self.task_config = task_config
 
         self.logger = getLogger(__name__)
```

Comment:
it will be good idea to maintain the same format across different component like either using self.logger or logger.

Reply:
Good catch. I missed refactoring `self.logger` for collaborator. Will do